content/docs/start/data-pipelines.md

```diff
@@ -1,13 +1,23 @@
+---
+description: 'Learn how DVC data pipelines can help you organize, version, and
+efficiently reproduce your data science or machine learning project.'
```

Comment:
```suggestion
reproduce your data science or machine learning project efficiently.'
```

Reply:
But do we need that term? "efficient"

Reply:
data science or machine learning project -> data science and machine learning processes?

Or some other wording that makes it specifically about the aspect of codifying pipelines (as dvc.yaml files). "Project" seems too generic.

Reply:
Reworked this, PTAL.

Reply:
Also added a meta title, keeping with the pattern for other Getting Started and Tutorial docs.

Reply:
OK thanks.